### ISSUE TITLE
Remove templating code from example config files in the docs

### ISSUE THREAD

**Issue by joelostblom** [ucbds-infra/otter-grader#540]
**Describe the bug**
The example config files at https://otter-grader.readthedocs.io/en/latest/workflow/otter_generate/container_image.html#environment-yml include templating code that should not be in the file the user creates (e..g `{% if not other_environment %}name: {{ otter_env_name }}` should just be `name: otter_env`)

![image](https://user-images.githubusercontent.com/4560057/194714655-09e8fbd2-c73c-42cd-980a-9fa19e97e447.png)

**Expected behavior**
The files should only include the relevant info so that we can copy and paste their content. Copy and pasting the current content with the templating lines causes `otter assign` to raise an error since the yaml format is not valid:

```
  raise ScannerError("while scanning for the next token", None,
yaml.scanner.ScannerError: while scanning for the next token
found character '%' that cannot start any token
  in "<unicode string>", line 1, column 2:
    {% if not other_environment %}nam ... 
```

**Comment by chrispyles** [comment]
This is a good idea and should just take some fiddling with sphinx.

**Comment by chrispyles** [comment]
With the recent switch to conda for environment management, the only template left in this page is the `setup.sh` file. Will need to figure out a clean way of rendering that template. https://otter-grader.readthedocs.io/en/beta/workflow/otter_generate/container_image.html

**Comment by chrispyles** [comment]
Fixed in #569
